src/plugins/packages/stacked-barchart/src/module.test.tsx

```diff
@@ -0,0 +1,330 @@
+import React from 'react';
```

Comment:
For your tests I highly suggest you to use [React testing library](https://testing-library.com/docs/react-testing-library/intro/).
You will not have to deal any more with how to render your component, how to clean the DOM, casting your `container` as a `HTMLDivElement`.

If you tests will grow or if you will create other plugins in the future you should maybe take a look at this lib and maybe use it.

Reply:
Great Idea! Thanks) Updated the tests using react testing library)